### DISCUSSION TITLE
Membership Application

### DISCUSSION THREAD

**Discussion by aljesusg** [kiali/kiali#6014] [Governance]
Hi forks!

After some months of contributions, @ferhoyos did the following job:

- kiali/kiali => 10 PR merged
- kiali/core-ui => 5 PR merged
- kiali/openshift-servicemesh-plugin => 2 PR merged
- issues reports and code reviews.

And he has other in pending to be merge.

I would like to nominate him to have more responsibilities for applying as a maintainer.

Governance mandates that voting should be given withing 7 working days. 

What do you think ?

**Comment by jshaughn** [comment]
Also, I am wondering whether we should re-categorize Xunzhuo to emeritus?  He has not been active for quite some time.  I'm fine leaving it alone, I wasn't sure how we wanted to handle maintainers that have not been active but have not declared they are leaving the project.

**Reply by aljesusg** [reply]
Totally agree, we should keep active maintainers

**Reply by Xunzhuo** [reply]
Hi team, please see comment in: https://istio.slack.com/archives/CJVRXB09F/p1681784899048059

**Comment by ferhoyos** [comment]
Thanks @aljesusg for the proposal and the rest of the team for your support! It is an honor to become part of kiali maintainer team and I will do my best to contribute to this amazing open source project.
